### PR TITLE
Add LTS 19.8

### PR DESCRIPTION
**Pull request by eborden**
This LTS has the latest `freckle-app` release with updated PG,
scientist, memcached, and bugsnag support.

### CODE REVIEW

**Review by pbrisbin — COMMENTED** [review]
lts-8 is out, if you want to avoid the FR.

**Comment by eborden** [comment]
Good call!

File: lts/19/8.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  - github: freckle/generic-arbitrary
+    commit: e183c7a5b60a78b80b9e35d1bbb76e676ce35d11
+
+  # === Dependencies we don't own
+  - bugsnag-1.0.0.1
+  - bugsnag-wai-1.0.0.1
+  - bugsnag-yesod-1.0.0.1
+  - country-0.2.2
+  - datadog-0.3.0.0
+  - ekg-core-0.1.1.7
```
Comment (pbrisbin) [inline lts/19/8.yaml:22]: Totally tangential, but: I continue to have zero idea how this works. The Hackage version of this dependency doesn't work (`base < 4.15`), it's why we have the fork we use elsewhere. And yet, here? It's fine.

**Review by pbrisbin — APPROVED** [review]

File: lts/19/8.yaml (hunk 1 above)
Comment (eborden) [reply lts/19/8.yaml:22]: `ekg-core`?

**Comment by eborden** [comment]
RCL link FYI: https://rcl.freckle.com/?from=lts-19.6&to=lts-19.8

**Comment by eborden** [comment]
This built locally with `tts` and `fancy` :+1:

File: lts/19/8.yaml (hunk 1 above)
Comment (pbrisbin) [reply lts/19/8.yaml:23]: Yes.

**Pull request merged**